### PR TITLE
Specify LD_LIBRARY_PATH as does start-tor-browser sh script

### PR DESCRIPTION
**Pull request by fowlslegs**

### CODE REVIEW

File: tbselenium/tbdriver.py [hunk 1]
```diff
@@ -161,10 +161,10 @@ def export_lib_path(self):
         tbb_root_dir = dirname(dirname(self.tbb_fx_binary_path))
         tbb_browser_dir = join(tbb_root_dir, cm.DEFAULT_TBB_BROWSER_DIR)
         tor_binary_dir = join(tbb_root_dir, cm.DEFAULT_TOR_BINARY_DIR)
-        # Add "TBB_DIR/Browser" and dir. of the tor binary to LD_LIBRARY_PATH
+        # Add "TBB_DIR/Browser/Tor" (the Tor binary dir) to LD_LIBRARY_PATH
+        # following the start-tor-browser script
         chdir(tbb_browser_dir)
-        environ["LD_LIBRARY_PATH"] = "%s:%s" % (tbb_browser_dir,
-                                                tor_binary_dir)
+        environ["LD_LIBRARY_PATH"] = tbb_binary_dir
```
Comment (gunesacar) [inline tbselenium/tbdriver.py]: I guess it should be `tor_binary_dir`

**Comment by gunesacar** [comment]
Looks great, except a minor issue. I added a comment.
Thanks!

**Comment by fowlslegs** [comment]
Do you want me to rebase somehow on this, or is this good to go?

**Comment by gunesacar** [comment]
I think I can take care of it. Let me look into it.

**Comment by fowlslegs** [comment]
Okay, let me know :smile_cat:

**Comment by gunesacar** [comment]
Would be great if you can rebase or just send a different PR based on the current master.
Sorry, :crying_cat_face:.

**Pull request closed without merging**